app/code/Magento/Customer/Block/Adminhtml/Edit/BackButton.php

```diff
@@ -32,6 +52,12 @@ public function getButtonData()
      */
     public function getBackUrl()
     {
+        if ($fromPath=$this->decoder->decode($this->form->getRequest()->getParam('fromPath'))) {
```

Comment:
Instead of adding this parameter from the request, can't we use the `RedirectInterface` and the method `getRefererUrl`? This should return the URL that the request came from.

Reply:
@gabrieldagama no, this will cause redirect to any previously opened page and works as the 'back' button in the browser.